src/jobs/generation/MetaGenerator.groovy

```diff
@@ -429,6 +429,8 @@ repos.each { repoInfo ->
                 if (isVSTS) {
                     stringParam('VSTSCollectionName', repoInfo.collection, 'VSTS collection name')
                     stringParam('VSTSCredentialsId', repoInfo.credentials, 'VSTS credentials id')
+                    stringParam('VSTSRefspec', vstsRefspec, 'VSTS refspec')
```

Comment:
This is incorrect.  The code above is referencing the VSTSRefSpec parameter, that is defined here, but injected by your plugin.  The second parameter here should probably be empty (default refspec).

Also, parameters are case sensitive, so be sure to match the name of this parameter with what your plugin is injecting